### PR TITLE
Add thumb up and down icon

### PR DESCRIPTION
**Pull request by AllanOXDi**
## Description
- Add thump up/down icon to support search recommendation work

### Before/after screenshots
Screenshot 2024-03-29 at 21 21 49

## Changelog

- [#587]
  - **Description:** Adds thumps down icon that is needed in Studio for search recomendation work.
  - **Products impact:** New Icon
  - **Addresses:** [#4450)](https://github.com/learningequality/studio/issues/4450)
  - **Components:** N/A
  - **Breaking:** No
  - **Impacts a11y:**  No
  - **Guidance:** 
[#603]: https://github.com/learningequality/kolibri-design-system/pull/603

## Testing checklist

- [x] Contributor has fully tested the PR manually
- [ ] If there are any front-end changes, before/after screenshots are included
- [ ] Critical and brittle code paths are covered by unit tests
- [ ] The change is described in the changelog section above

## Reviewer guidance

- [ ] Is the code clean and well-commented?
- [ ] Are there tests for this change?
- [ ] Are all UI components LTR and RTL compliant (if applicable)?
- [ ] _Add other things to check for here_

## After review

- [ ] The changelog item has been pasted to the `CHANGELOG.md`

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,16 @@ Changelog is rather internal in nature. See release notes for the public overvie
 
 ## Version 3.x.x (`release-v3` branch)
 
+- [#587]
```
Comment (MisRob) [inline CHANGELOG.md]: This needs to be #603

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,16 @@ Changelog is rather internal in nature. See release notes for the public overvie
...
+  - **Description:** Adds thumps down icon that is needed in Studio for search recomendation work.
+  - **Products impact:** New Icon
+  - **Addresses:** [#4450)](https://github.com/learningequality/studio/issues/4450)
```
Comment (MisRob) [inline CHANGELOG.md]: A small typo - could you remove ")"?

```suggestion
  - **Addresses:** [#4450](https://github.com/learningequality/studio/issues/4450)
```

**Review by MisRob — APPROVED** [review]
Thank you, @AllanOXDi, it's looking good. Would you fix the typos before merging?

We can leave this PR as is and for future icons, please note that we're migrating away from using `./precompiled-icons/material-icons` in favor of pasting svgs from Figma to `custom-icons` (https://github.com/learningequality/kolibri-design-system/issues/540). It's described in this guide https://github.com/learningequality/kolibri-design-system/blob/release-v4/dev_docs/05_icons.md.

File: CHANGELOG.md (hunk 2 above)
Comment (AllanOXDi) [reply CHANGELOG.md]: Good catch . Thank you, fixed!

**Pull request merged**